### PR TITLE
MAWB 6

### PR DESCRIPTION
**Pull request by DmitriyLitvin**

### CODE REVIEW

**Comment by boroda4436** [comment]
@BondarenkoJek Please investigate why Trevis CI check wasn't applied

File: src/main/java/mate/academy/wikibot/http/YouTubeRequestController.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+     *
+     * @param data
+     * @return list of YouTubeResponseDto
+     */
+    private static List<YouTubeResponseDto> deserializeJson(String data) {
+        ObjectMapper objectMapper = new ObjectMapper();
+        List<YouTubeResponseDto> responseDtos = new ArrayList<>();
+
+        try {
+            JsonNode rootNode = objectMapper.readTree(data);
```
Comment (boroda4436) [inline src/main/java/mate/academy/wikibot/http/YouTubeRequestController.java]: @DmitriyLitvin @romaanankin we need YoutubeVideoListResponseDto

**Comment by Dinexpod** [comment]
For me looks as good.

File: .travis.yml [hunk 2]
```diff
@@ -1,12 +0,0 @@
-language: java
```
Comment (YatKat) [inline .travis.yml]: Deleting or tis file will delete it from master after merging. It's not good

File: src/main/java/mate/academy/wikibot/http/YouTubeRequestController.java [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+                        + "&fields=items(id%%2FvideoId%%2Csnippet%%2Ftitle)"
+                        + "&key=%s",
+                requestDto.getMaxResults(),
+                requestDto.getQuery(),
+                requestDto.getTopicId(),
+                requestDto.getApiKey());
+
+        try {
+            HttpClient client = ClientHandler.getHttpClientInstance();
+            StringBuilder stringBuilder = new StringBuilder(url);
```
Comment (YatKat) [inline src/main/java/mate/academy/wikibot/http/YouTubeRequestController.java]: Why you use StringBuilder if at the next line you make url again toString.

**Pull request merged**